tardis/tests/integration_tests/plot_helpers.py

```diff
@@ -2,7 +2,8 @@
 import tempfile
 
 from pytest_html import extras
-from tardis import __githash__ as tardis_githash
+
+# from tardis import __githash__ as tardis_githash
```

Comment:
Why comment and not just delete?

Reply:
Someone will need to fix integration tests in the future and maybe that line would be useful. I was thinking of adding a comment like `not available in version.py anymore, find another way!` or something like that.